tests/test_client.py

```diff
@@ -992,3 +992,31 @@ async def test_async_client_create_blob_exists(httpserver: HTTPServer):
   with tempfile.NamedTemporaryFile() as blob:
     response = await client._create_blob(blob.name)
     assert response == 'sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+
+
+def test_parse_host(httpserver: HTTPServer):
```

Comment:
this is unnecessary since these are tested in the doc test

Reply:
Ah thanks, I missed that!